### PR TITLE
Create a nginx for local dev and add configuration for using stomp be…

### PR DESCRIPTION
**Pull request by titus6012000**
…tween the front-end and the back-end. The task-notification-server becomes useless.

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [X] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
yes

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
new feature, and remove of useless feature

**What is the new behavior (if this is a feature change)?**
add configuration in order to use stomp between the front and the back. The notification-server becomes useless

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: docker-compose/cse-export-d2cc/config/cse-export-d2cc-runner-application.yml [hunk 1]
```diff
@@ -10,7 +10,18 @@ spring:
       reply-timeout: 7200000
   cloud:
     stream:
+      function:
+        definition: interrupt;request
```
Comment (TheMaskedTurtle) [inline docker-compose/cse-export-d2cc/config/cse-export-d2cc-runner-application.yml:14]: Can be removed

File: docker-compose/cse-export-d2cc/config/cse-export-d2cc-task-manager-application.yml [hunk 2]
```diff
@@ -62,3 +62,15 @@ minio-adapter:
...
+  allowed-origin: http://cse-export-d2cc-gridcapa-app
+  starting-ws-endpoint: /tasks/notify
+  notify: /task
+  receive-request: /app
+
+ftp:
+  ftp-host: ftp-server
+  ftp-port: 21
+  ftp-user: gridcapa
+  ftp-password: gridcapa
```
Comment (TheMaskedTurtle) [inline docker-compose/cse-export-d2cc/config/cse-export-d2cc-task-manager-application.yml:76]: What is that ?

File: docker-compose/cse-export-idcc/config/cse-export-idcc-runner-application.yml [hunk 3]
```diff
@@ -10,7 +10,18 @@ spring:
       reply-timeout: 7200000
   cloud:
     stream:
+      function:
+        definition: interrupt;request
```
Comment (TheMaskedTurtle) [inline docker-compose/cse-export-idcc/config/cse-export-idcc-runner-application.yml:14]: Same

File: docker-compose/cse-export-idcc/config/cse-export-idcc-task-manager-application.yml [hunk 4]
```diff
@@ -62,3 +62,16 @@ minio-adapter:
...
+  allowed-origin: http://cse-export-idcc-gridcapa-app
+  starting-ws-endpoint: /tasks/notify
+  notify: /task
+  receive-request: /app
+
+ftp:
+  ftp-host: ftp-server
+  ftp-port: 21
+  ftp-user: gridcapa
+  ftp-password: gridcapa
```
Comment (TheMaskedTurtle) [inline docker-compose/cse-export-idcc/config/cse-export-idcc-task-manager-application.yml:76]: Same

File: docker-compose/cse-import-d2cc/config/cse-d2cc-gridcapa-runner-application.yml [hunk 5]
```diff
@@ -10,7 +10,18 @@ spring:
       reply-timeout: 7200000
   cloud:
     stream:
+      function:
+        definition: interrupt;request
```
Comment (TheMaskedTurtle) [inline docker-compose/cse-import-d2cc/config/cse-d2cc-gridcapa-runner-application.yml:14]: Same

File: docker-compose/cse-import-d2cc/config/cse-d2cc-gridcapa-task-manager-application.yml [hunk 6]
```diff
@@ -65,4 +65,16 @@ minio-adapter:
...
+  allowed-origin: http://cse-import-d2cc-gridcapa-app
+  starting-ws-endpoint: /tasks/notify
+  notify: /task
+  receive-request: /app
+
+ftp:
+  ftp-host: ftp-server
+  ftp-port: 21
+  ftp-user: gridcapa
+  ftp-password: gridcapa
```
Comment (TheMaskedTurtle) [inline docker-compose/cse-import-d2cc/config/cse-d2cc-gridcapa-task-manager-application.yml:80]: Same

File: docker-compose/cse-import-idcc/config/cse-idcc-runner-application.yml [hunk 7]
```diff
@@ -10,7 +10,18 @@ spring:
       reply-timeout: 7200000
   cloud:
     stream:
+      function:
+        definition: interrupt;request
```
Comment (TheMaskedTurtle) [inline docker-compose/cse-import-idcc/config/cse-idcc-runner-application.yml:14]: Same

File: docker-compose/cse-import-idcc/config/cse-idcc-task-manager-application.yml [hunk 8]
```diff
@@ -71,4 +71,16 @@ minio-adapter:
...
+  allowed-origin: http://cse-import-idcc-gridcapa-app
+  starting-ws-endpoint: /tasks/notify
+  notify: /task
+  receive-request: /app
+
+ftp:
+  ftp-host: ftp-server
+  ftp-port: 21
+  ftp-user: gridcapa
+  ftp-password: gridcapa
```
Comment (TheMaskedTurtle) [inline docker-compose/cse-import-idcc/config/cse-idcc-task-manager-application.yml:86]: Same

File: k8s/overlays/azure/test/gridcapa-ingress.yaml [hunk 9]
```diff
@@ -52,14 +52,15 @@ spec:
...
+                name: core-valid-task-manager
                 port:
                   number: 80
             path: /core/valid/task-notification(/|$)(.*)
             pathType: Prefix
           - backend:
               service:
-                name: core-valid-task-manager
+                name: core-valid-  server:
+    port: 8081
```
Comment (TheMaskedTurtle) [inline k8s/overlays/azure/test/gridcapa-ingress.yaml:63]: I think there is an error here

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]

File: docker-compose/cse-export-d2cc/config/cse-export-d2cc-runner-application.yml (hunk 1 above)
Comment (titus6012000) [reply docker-compose/cse-export-d2cc/config/cse-export-d2cc-runner-application.yml:14]: removed

File: docker-compose/cse-export-d2cc/config/cse-export-d2cc-task-manager-application.yml (hunk 2 above)
Comment (titus6012000) [reply docker-compose/cse-export-d2cc/config/cse-export-d2cc-task-manager-application.yml:76]: the future...
i remove it for now

File: docker-compose/cse-export-idcc/config/cse-export-idcc-runner-application.yml (hunk 3 above)
Comment (titus6012000) [reply docker-compose/cse-export-idcc/config/cse-export-idcc-runner-application.yml:14]: removed

File: docker-compose/cse-export-idcc/config/cse-export-idcc-task-manager-application.yml (hunk 4 above)
Comment (titus6012000) [reply docker-compose/cse-export-idcc/config/cse-export-idcc-task-manager-application.yml:76]: remove until it come back later

File: docker-compose/cse-import-d2cc/config/cse-d2cc-gridcapa-runner-application.yml (hunk 5 above)
Comment (titus6012000) [reply docker-compose/cse-import-d2cc/config/cse-d2cc-gridcapa-runner-application.yml:14]: removed

File: docker-compose/cse-import-d2cc/config/cse-d2cc-gridcapa-task-manager-application.yml (hunk 6 above)
Comment (titus6012000) [reply docker-compose/cse-import-d2cc/config/cse-d2cc-gridcapa-task-manager-application.yml:80]: remove until it comes back later

File: docker-compose/cse-import-idcc/config/cse-idcc-runner-application.yml (hunk 7 above)
Comment (titus6012000) [reply docker-compose/cse-import-idcc/config/cse-idcc-runner-application.yml:14]: removed

File: docker-compose/cse-import-idcc/config/cse-idcc-task-manager-application.yml (hunk 8 above)
Comment (titus6012000) [reply docker-compose/cse-import-idcc/config/cse-idcc-task-manager-application.yml:86]: remove until it comes back later

File: k8s/overlays/azure/test/gridcapa-ingress.yaml (hunk 9 above)
Comment (titus6012000) [reply k8s/overlays/azure/test/gridcapa-ingress.yaml:63]: I roll back this piece of ...

**Review by TheMaskedTurtle — APPROVED** [review]

**Review by TheMaskedTurtle — APPROVED** [review]

**Review by TheMaskedTurtle — APPROVED** [review]

**Pull request merged**